python_modules/libraries/dagster-sling/dagster_sling/asset_decorator.py

```diff
@@ -57,6 +56,7 @@ def sling_assets(
     replication_config: SlingReplicationParam,
     dagster_sling_translator: Optional[DagsterSlingTranslator] = None,
     name: Optional[str] = None,
+    group_name: Optional[str] = None,
```

Comment:
Maybe we don't want to add the feature here though? Because `group_name` was not passed to the asset decorator?